websites/management/commands/update_departments.py

```diff
@@ -0,0 +1,33 @@
+from functools import reduce
+
+from django.core.management import BaseCommand
+from django.db import transaction
+from django.db.models import Q
+
+from websites.models import WebsiteContent
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        filter_set = ["21W", "CMS"]
+        filter_query_set = [
+            "website__name__istartswith",
+            "website__short_id__istartswith",
+        ]
+
+        query_set = WebsiteContent.objects.filter(
+            reduce(
+                lambda x, y: x | y,
+                [Q(**{key: value}) for key, value in zip(filter_query_set, filter_set)],
+            ),
+        )
+
+        query_set = query_set.filter(type="sitemetadata")
+        with transaction.atomic():
+            for sitemetadata in query_set.iterator():
+                if not sitemetadata.metadata["department_numbers"]:
+                    sitemetadata.metadata["department_numbers"] = [
+                        *sitemetadata.metadata["department_numbers"],
+                        "CMS-W",
+                    ]
+                    sitemetadata.save()
```

Comment:
Might be useful to have some kind of feedback for the command, like "X sites were processed", maybe with a list of affected sites.  Alternately, since this command is likely to be run only once, maybe a migration would be more suitable?

```python
from functools import reduce

from django.db import migrations
from django.db import transaction
from django.db.models import Q


def set_cms_w_department(apps, schema_editor):
    """ Update the CMS department"""
    WebsiteContent = apps.get_model("websites", "WebsiteContent")
    filter_set = ["21W", "CMS"]
    filter_query_set = [
        "website__name__istartswith",
        "website__short_id__istartswith",
    ]

    query_set = WebsiteContent.objects.filter(
        reduce(
            lambda x, y: x | y,
            [Q(**{key: value}) for key, value in zip(filter_query_set, filter_set)],
        ),
    )

    query_set = query_set.filter(type="sitemetadata")
    with transaction.atomic():
        for sitemetadata in query_set.iterator():
            if not sitemetadata.metadata["department_numbers"]:
                sitemetadata.metadata["department_numbers"] = [
                    *sitemetadata.metadata["department_numbers"],
                    "CMS-W",
                ]
                sitemetadata.save()


class Migration(migrations.Migration):

    dependencies = [
        ("websites", "0045_blank_publish_fields"),
    ]

    operations = [
        migrations.RunPython(set_cms_w_department, migrations.RunPython.noop),
    ]
```